docroot/themes/custom/mass_theme/templates/content/node--info-details.html.twig

```diff
@@ -122,7 +111,6 @@
         {% endblock %}
       {% endembed %}
     {% endif %}
-  {% endif %}
```

Comment:
Since an outer if() statement if removed, I would have expected all these lines to have 2 spaces less of indent. But the lines did not move. Maybe the diff is misleading.

Reply:
it's correct. I don't see any spacing issues.

Reply:
I pushed a commit which removed 2 spaces of indentation.

Reply:
Strange, it was not showing in my IDE.